lib/app-express.js

```diff
@@ -106,11 +169,31 @@ module.exports = function (app) {
     return res.redirect(uri);
   });
 
+  app.get('/hubLinked', (req, res) => {
+    res.render('pages/hubLinked', {
+      username : req.cookies.user_name
+    });
+  });
+
+  app.get('/permanentLogin', (req, res) => {
+    const template = req.query.template;
+
+    const uri = oauth2.getAuthorizationUrl({
+      redirect_uri: callbackUrl,
+      client_id: consumerKey,
+      scope: 'id api openid refresh_token',
+      state: template,
+      prompt: 'select_account'
+    });
+
+    return res.redirect(uri);
+  });
+
   app.get('/logout', (req, res) => {
     const access_token = req.cookies.access_token;
     const instance_url = req.cookies.instance_url;
 
-    const conn = new jsforce.Connection({
+    conn = new jsforce.Connection({
```

Comment:
Should be `const`.

Reply:
fixed, as a new local const.  I'm using *conn* as the global for the hosted hub (set from init or from cache).